### PR TITLE
[MINOR] Update logging lib, messages and implementation.

### PR DESCRIPTION
**Pull request by vdaviot**

### CODE REVIEW

File: script.py [hunk 1]
```diff
@@ -3,30 +3,54 @@
 import os
 import time
 import logging
+import structlog
```
Comment (arobase-che) [inline script.py:6]: 👍

File: script.py [hunk 2]
```diff
@@ -101,6 +123,8 @@ def get_secret_values(self, secret_name):
             # https://docs.aws.amazon.com/secretsmanager/latest/apireference/API_GetSecretValue.html
             raise e
 
+        self.logger.info(msg="Found secret", secret_name=secret_name)
```
Comment (rmasclef) [inline script.py:126]:
```suggestion
        self.logger.info(msg="Fecret found", secret_name=secret_name)
```

+you don't want to pass the k8s_namespace ?

File: script.py [hunk 3]
```diff
@@ -113,59 +137,76 @@ def get_secret_namespace_tag(self, aws_secret):
...
+                labels={self.params['aws_tag_key']: self.params['aws_tag_value']}
+            ),
             data=data
         )
+
         try:
-            # Check if the secret already exists
-            existing_data = self.v1_api.read_namespaced_secret(name=name, namespace=namespace)
+            # Check if the secret exists
+            existing_data = self.v1_api.read_namespaced_secret(name=secret_name, namespace=namespace)
```
Comment (rmasclef) [inline script.py:154]:
```suggestion
            existing_data = self.v1_api.read_namespaced_secret(name=secret_name, k8s_namespace=namespace)
```

File: script.py (hunk 2 above)
Comment (vdaviot) [reply script.py:141]: we don't have it there

File: script.py [hunk 4]
```diff
@@ -3,41 +3,81 @@
...
 import logging
+import structlog
 import boto3
 from botocore.exceptions import ClientError
 from kubernetes import client, config
 
+AWS_REGION = os.environ['AWS_REGION']
+
+APPLICATION_NAME = 'aws-secrets-synchronizer'
+APPLICATION_VERSION =  os.environ['APPLICATION_VERSION'] # Mandatory, will raise KeyError if not set
```
Comment (fabienduhamel) [inline script.py:14]: must be a constant, not an env var

File: script.py [hunk 5]
```diff
@@ -3,41 +3,81 @@
...
+ENVIRONMENT_TYPE = os.environ['ENVIRONMENT_TYPE'] # Mandatory, will raise KeyError if not set
+ENVIRONMENT_NAME = os.environ.get('ENVIRONMENT_NAME')
```
Comment (fabienduhamel) [inline script.py:16]: These 2 could be removed.

**Review by fabienduhamel — CHANGES_REQUESTED** [review]

File: script.py (hunk 4 above)
Comment (vdaviot) [reply script.py:14]: whatever TBH, i'd like to get it from pyproject.toml directly

File: script.py (hunk 5 above)
Comment (vdaviot) [reply script.py:16]: Those are juste example variable, tho i think ENVIRONMENT_TYPE remains important

File: script.py (hunk 4 above)
Comment (fabienduhamel) [reply script.py:14]: Idk how but okay

File: script.py (hunk 5 above)
Comment (fabienduhamel) [reply script.py:16]: It is already populated by our collector.

**Pull request merged**